.github/workflows/build.yml

```diff
@@ -0,0 +1,212 @@
+      - run: cd secrethub && Rename-Item -Path "Client.dll" -NewName "CLIENT.dll"
+      - run: python -m pip install --upgrade setuptools wheel
+      - run: choco install swig
+      - run: python setup.py bdist_wheel --plat-name=win32
+        if: ${{ matrix.arch == 'x86' }}
+      - run: python setup.py bdist_wheel --plat-name=win-amd64
+        if: ${{ matrix.arch == 'x64' }}
+      - run: mkdir wheelhouse
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+          architecture: ${{ matrix.arch }}
+      - run: python -m pip install pefile machomachomangler
+      - run: python utilities/wheel-fix.py ./dist/*.whl ./secrethub/CLIENT.dll ./wheelhouse
+        shell: bash
+      - uses: actions/upload-artifact@v2
+        with:
+          name: secrethub-python-${{ matrix.python-version }}-windows-2019-${{ matrix.arch }}
+          path: wheelhouse
+  test:
+    needs: [build-linux-client, build-windows-client, build-macos-client]
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, windows-2019, macos-10.14, macos-10.15]
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
+        arch: ['x64']
+        include:
+          - os: windows-2019
+            python-version: 3.5
+            arch: 'x86'
+          - os: windows-2019
+            python-version: 3.6
+            arch: 'x86'
+          - os: windows-2019
+            python-version: 3.7
+            arch: 'x86'
+          - os: windows-2019
+            python-version: 3.8
+            arch: 'x86'
+          - os: windows-2019
+            python-version: 3.9
+            arch: 'x86'
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: false
+      - run: rm -rf secrethub
+        shell: bash
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: ${{ matrix.arch }}
+      - uses: actions/download-artifact@v2
+        with:
+          name: secrethub-python-${{ matrix.python-version }}-${{ matrix.os }}
+          path: .
+        if: ${{ matrix.os != 'windows-2019' }}
+      - uses: actions/download-artifact@v2
+        with:
+          name: secrethub-python-${{ matrix.python-version }}-${{ matrix.os }}-${{ matrix.arch }}
+          path: .
+        if: ${{ matrix.os == 'windows-2019' }}
+      - uses: ilammy/msvc-dev-cmd@v1
+        with:
+          arch: ${{ matrix.arch }}
+        if: ${{ matrix.os == 'windows-2019' }}
+      - run: python -m pip install --upgrade pip
+      - run: python -m pip install *.whl
+        shell: bash
+      - run: SECRETHUB_TEST_DIR=secrethub/xgo/python/test python -m unittest test/test.py
+        shell: bash
+        env:
+          SECRETHUB_CREDENTIAL: ${{ secrets.TEST_SECRETHUB_CREDENTIAL }}
+  publish:
+    if: github.event_name != 'push' && github.ref == 'refs/heads/master'
```

Comment:
This means wer're publishing every commit on `master`. What happens if the version in `setup.py` is not updated and we merge a change to `master`? Does that mean we re-release an old version with a change?

Reply:
No. Twine currently fails if you try to upload a file that was already published to pypi.

Reply:
The actual branch is currently called `main`, btw. So this is not going to work :grin:

Reply:
Oops :sweat_smile: